### DISCUSSION TITLE
Refresh/Reload Icon

### DISCUSSION THREAD

**Discussion by prabhat-hbo** [tailwindlabs/heroicons#741] [Ideas]
It would be nice to have a Refresh/Reload icon.

**Comment by LarsFlieger** [comment]
Maybe you can use `arrow-path`:
![image](https://user-images.githubusercontent.com/47084064/186448855-ebf05a9c-b829-464c-b60b-c0112f628007.png)

**Reply by bill-offertrackr** [reply]
Usually I animate a refresh icon by rotating it to show the user when something is refreshing. This worked great in v1 since the arrow path was circular, the animation looked really smooth.  In v2, doing something similar with arrow-path which is squarish leads to a really clunky animation.

**Reply by abcmoritz** [reply]
Really good point!

**Reply by sschoger** [reply]
Hey! We just added a circular `arrow-path` icon like the v1 that should support this now 👍🏻
<img width="347" alt="CleanShot 2022-08-30 at 14 28 02@2x" src="https://user-images.githubusercontent.com/2652515/187515058-962e4080-286f-4a37-af8c-a39eaa80e325.png">
